src/Search.js

```diff
@@ -26,4 +26,8 @@ const Search = () => {
   );
 };
 
+const SearchButton = () => {
```

Comment:
It's a good idea to extract this component into its own file.

Also I'd like you to have a think about the following:

- What if we made this component even more generic so that we could pass in any text in props?
- The component would have manage styling and the text for the component would come from a parent component
- Would we still have to call this `SearchButton` then?
- Think about how we could we implement this and what benefits this has!

Reply:
the first part I understood and done but the last four point that you mentioned did not understand.